modules/apps/object/object-rest-test/src/testFunctional/tests/ManageCategoriesForObjectEntries.testcase

```diff
@@ -153,6 +292,52 @@ definition {
 		}
 	}
 
+	@disable-webdriver = "true"
+	@priority = 3
+	test CanUpdateSiteScopedObjectEntryWithTaxonomyCategoryOfDifferentStiteScope {
```

Comment:
Typo? Please update the name to be end with SiteScope instead of StiteScope :smile:

Reply:
Typo appears in test scenarios, I should modify it in the test scenarios and the test map